classes/api.php

```diff
@@ -0,0 +1,112 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace auth_saml2;
+
+defined('MOODLE_INTERNAL') || die();
+
+use moodle_url;
+
+/**
+ * Static list of api methods for auth saml2 configuration.
+ *
+ * @package   auth_saml2
+ * @copyright 2021 Moodle Pty Ltd <support@moodle.com>
```

Comment:
If things are mostly just a refactor of existing code can you please add or retail copyright for 'Catalyst IT' please

Reply:
OK